bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/handler/VeluxBridgeHandler.java

```diff
@@ -631,27 +650,27 @@ private synchronized void handleCommandScheduled(ChannelUID channelUID, Command
 
                         default:
                             logger.trace(
-                                    "handleCommandScheduled(): cannot handle REFRESH on channel {} as it is of type {}.",
+                                    "handleCommandCommsJob(): cannot handle REFRESH on channel {} as it is of type {}.",
                                     itemName, channelId);
                     }
                 } catch (IllegalArgumentException e) {
                     logger.warn("Cannot handle REFRESH on channel {} as it isn't (yet) known to the bridge.", itemName);
                 }
                 if (newState != null) {
                     if (itemType.isChannel()) {
-                        logger.debug("handleCommandScheduled(): updating channel {} to {}.", channelUID, newState);
+                        logger.debug("handleCommandCommsJob(): updating channel {} to {}.", channelUID, newState);
                         updateState(channelUID, newState);
                     } else if (itemType.isProperty()) {
                         // if property value is 'unknown', null it completely
                         String val = newState.toString();
                         if (VeluxBindingConstants.UNKNOWN.equals(val)) {
                             val = null;
                         }
-                        logger.debug("handleCommandScheduled(): updating property {} to {}.", channelUID, val);
+                        logger.debug("handleCommandCommsJob(): updating property {} to {}.", channelUID, val);
                         ThingProperty.setValue(this, itemType.getIdentifier(), val);
                     }
                 } else {
-                    logger.info("handleCommandScheduled({},{}): updating of item {} (type {}) failed.",
+                    logger.info("handleCommandCommsJob({},{}): updating of item {} (type {}) failed.",
```

Comment:
Actually this is an error situation, so it should probably have been a 'warn' rather than an 'info' even; but I suggest we leave it as an 'info' as it was before..

Reply:
As this seems to be an unrecoverable error, I'd go for warn, too.

Reply:
:)